proxy_map.go

```diff
@@ -1348,7 +1347,7 @@ func (m *Map) LocalMapStats() LocalMapStats {
 	return LocalMapStats{}
 }
 
-func (m *Map) destroyLocally(ctx context.Context) {
+func (m *Map) destroyLocally(ctx context.Context) bool {
```

Comment:
it always returns true, why return a `bool` ?

Reply:
No need, updated at: https://github.com/hazelcast/hazelcast-go-client/pull/900/commits/22016a7a134f0e91e54acda75e73cb1e1e063a52